### PR TITLE
Fix qedma examples

### PR DESCRIPTION
**Pull request by assafb**

### CODE REVIEW

File: docs/guides/qedma-qesem.ipynb [hunk 1]
```diff
@@ -78,9 +78,9 @@
    "source": [
     "from qiskit_ibm_catalog import QiskitFunctionsCatalog\n",
     "\n",
-    "catalog = QiskitFunctionsCatalog(instance=\"\")\n",
+    "catalog = QiskitFunctionsCatalog(token = \"<YOUR_IQP_API_TOKEN>\")\n",
```
Comment (Eric-Arellano) [inline docs/guides/qedma-qesem.ipynb:81]:
```suggestion
    "catalog = QiskitFunctionsCatalog(token=\"<YOUR_IQP_API_TOKEN>\")\n",
```

File: docs/guides/qedma-qesem.ipynb [hunk 2]
```diff
@@ -108,17 +108,17 @@
    "outputs": [],
    "source": [
     "job = qesem_function.run(\n",
-    "    action = \"estimate_qpu_time\",\n",
+    "    action = \"estimate_qpu_time\", \n",
```
Comment (Eric-Arellano) [inline docs/guides/qedma-qesem.ipynb:111]: Nit, this change doesn't seem necessary

```suggestion
    "    action = \"estimate_qpu_time\",\n",
```

File: docs/guides/qedma-qesem.ipynb [hunk 3]
```diff
@@ -108,17 +108,17 @@
...
     "    instance = \"hub/group/project\",\n",
     "    pubs = [\n",
     "        (\n",
-    "            bell,\n",
+    "            circ, \n",
```
Comment (Eric-Arellano) [inline docs/guides/qedma-qesem.ipynb:115]:
```suggestion
    "            circ,\n",
```

File: docs/guides/qedma-qesem.ipynb [hunk 4]
```diff
@@ -138,19 +138,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# example execute QESEM job\n",
     "job = qesem_function.run(\n",
-    "    action = \"execute\",\n",
+    "    action = \"execute\", \n",
```
Comment (Eric-Arellano) [inline docs/guides/qedma-qesem.ipynb:142]:
```suggestion
    "    action = \"execute\",\n",
```

File: docs/guides/qedma-qesem.ipynb [hunk 5]
```diff
@@ -138,19 +138,18 @@
...
     "    instance = \"hub/group/project\",\n",
     "    pubs = [\n",
     "        (\n",
-    "            bell,\n",
+    "            circ, \n",
```
Comment (Eric-Arellano) [inline docs/guides/qedma-qesem.ipynb:146]:
```suggestion
    "            circ,\n",
```

**Review by Eric-Arellano — COMMENTED** [review]
Thanks! Once you sign the contributor license agreement (see the comment above mine), I'll merge this and deploy it live.

Don't worry about the red CI: it's expected because this is a fork.

**Comment by assafb** [comment]
> Thanks! Once you sign the contributor license agreement (see the comment above mine), I'll merge this and deploy it live.
> 
> Don't worry about the red CI: it's expected because this is a fork.

Thanks @Eric-Arellano . Signed!

**Comment by pandasa123** [comment]
@assafb, is it possible to add the `circ` you're using here? Just to have an end-to-end runnable example

**Comment by Eric-Arellano** [comment]
> is it possible to add the circ you're using here? Just to have an end-to-end runnable example

I'm going to merge this to get out the bug fix sooner. @assafb can you please open a follow-up PR with Sanket's request? Thanks!

**Pull request merged**